src/Bll/StatementBLL.php

```diff
@@ -356,6 +356,64 @@ public function acceptFundsById(int $statementId, $statementDto = null): int
         }
     }
 
+    /**
+     * @param int $statementId
+     * @param float $partialAmount
+     * @param StatementDTO|null $statementDto
+     * @return int
+     * @throws AccountException
+     * @throws AmountException
+     * @throws InvalidArgumentException
+     * @throws OrmBeforeInvalidException
+     * @throws OrmInvalidFieldsException
+     * @throws RepositoryReadOnlyException
+     * @throws StatementException
+     * @throws UpdateConstraintException
+     * @throws \ByJG\MicroOrm\Exception\InvalidArgumentException
+     */
+    public function acceptPartialFundsById(int $statementId, float $partialAmount, StatementDTO $statementDto = null): int
```

Comment:
I am thinking here. If we are passing the statementDto, maybe, the parameter $partialAmount can be suppressed . What is your opinion about It?